### ISSUE TITLE
Gazebo and Ipopt symbol collision problem

### ISSUE THREAD

**Issue by traversaro** [robotology/cer-sim#12]
I do not know if the problem reported in https://bitbucket.org/osrf/gazebo/issues/2005/segfault-when-calling-ipopt-virtual-method is still a problem for the R1 simulator, but given I was blocked on that in another context, I gave it a shot an I generated drop-in replacements for the simbody packages in Ubuntu 18.04 . The packages are available in 
[simbody-ubuntu-1804-ipopt-fix-dropin.zip](https://github.com/robotology/cer-sim/files/2544918/simbody-ubuntu-1804-ipopt-fix-dropin.zip). It should be sufficient to just extract the archive and install the `libsimbody3.5v5_3.5.4+dfsg2-1_amd64.deb` and the `libsimbody-dev_3.5.4+dfsg2-1_amd64.deb` packages with 
~~~
dpkg -i libsimbody3.5v5_3.5.4+dfsg2-1_amd64.deb
dpkg -i libsimbody-dev_3.5.4+dfsg2-1_amd64.deb
~~~

Related Gazebo comment: https://bitbucket.org/osrf/gazebo/issues/2005/segfault-when-calling-ipopt-virtual-method#comment-48717745 .

**Comment by traversaro** [comment]
cc @drdanz @aerydna @mbrunettini @barbalberto @pattacini

**Comment by traversaro** [comment]
Similar packages but for Ubuntu 16.04 : 
[simbody-ubuntu-1604-ipopt-fix-dropin.zip](https://github.com/robotology/cer-sim/files/2545660/simbody-ubuntu-1604-ipopt-fix-dropin.zip).

**Comment by pattacini** [comment]
This drop-in is working very nicely but it somehow messes up `apt`.

**Comment by traversaro** [comment]
> This drop-in is working very nicely but it somehow messes up `apt`.

In which sense? In the previous comment I actually forgot to add that you should also install the `libsimbody-dev` package as well, that could create problems.

**Comment by pattacini** [comment]
I was getting several warnings from `apt` to fix dependencies.
After I installed `libsimbody-dev`, they suddenly disappeared.

**Comment by traversaro** [comment]
Great! Let me know if you had other problems. Ideally we should be able to distribute these packages as part of some Debian/Ubuntu repo.

**Comment by traversaro** [comment]
As Simbody 3.6 (that contains a fix for this issue, see https://github.com/simbody/simbody/issues/510) is available in Ubuntu repos since 19.04 (see https://repology.org/project/simbody/versions), once we update to Ubuntu 20.04 we should finally get rid of this issue.

**Comment by randaz81** [comment]
fixed in Ubuntu 20.04
